README.md

```diff
@@ -12,8 +12,7 @@ The .NET [OpenTelemetry](https://opentelemetry.io/) client.
 
 ## Getting Started
 
-If you are new here, please [get started in 5 minutes](./docs/trace/getting-started.md).
-
+If you are new here, please [get started in 5 minutes](./docs/trace/getting-started/README.md).

```

Comment:
Seems like something is missing in that sentence? Either that or we're asking them to wait 5 minutes before getting started 🤣 

How about:

`If you are new here, please see [get started in 5 minutes](./docs/trace/getting-started/README.md).`

or just

`If you are new here, see [get started in 5 minutes](./docs/trace/getting-started/README.md).`

Reply:
#1061.